bcda/models/fhir/alr/coverage.go

```diff
@@ -0,0 +1,46 @@
+package alr
+
+import (
+	//"github.com/CMSgov/bcda-app/bcda/models"
+	//fhircodes "github.com/google/fhir/go/proto/google/fhir/proto/stu3/codes_go_proto"
```

Comment:
nitpicking uneeded comments :)
also, confirming that r4 update will be in a separate pr?